test/libsolidity/syntaxTests/constants/math_functions_opcodes_pure_assignment_error.sol

```diff
@@ -0,0 +1,12 @@
+contract A {
+    uint256 k = 7;
+    uint256 constant amod = addmod(1, 8, k);
+    uint256 constant mmod = mulmod(1, 8, k);
```

Comment:
Do we have a variant of this test but with constant args?

Reply:
Regarding the former, we do have in https://github.com/argotorg/solidity/blob/e04cedafc509e2b56f311e2478b3413ca94bcd20/test/libsolidity/syntaxTests/constants/addmod_zero.sol.
Added for the later.

Reply:
Done